download-scopus-publications/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Har vi ikke cloudWatch som logger for oss? Bruker vi fortsatt log4j? Har vi gjort noen avgjørelse rundt det i det hele tatt?

Reply:
Det er i boilerplaten vår, det er Lambda-appender og pattern slik at det er likt.